ext/lists.go

```diff
@@ -65,6 +90,26 @@ func (listsLib) CompileOptions() []cel.EnvOption {
 				}),
 			),
 		),
+		cel.Function("flatten",
```

Comment:
@fabriziosestito Thanks for the feature updates! When adding new functions you'll need to introduce a concept of versions to the library. See `ext/strings.go` as an example of how this should look. The idea is that users should be able to opt-in to the new version or remain frozen.